packages/esm-ward-app/src/config-schema.ts

```diff
@@ -133,4 +135,4 @@ export interface PatientAddressElementConfig {
   addressFields: Array<keyof PersonAddress>;
 }
 
-export type PatientCardElementConfig = {} & PatientAddressElementConfig;
+export type PatientCardElementConfig = { identifierTypeUuid: string } & PatientAddressElementConfig;
```

Comment:
Let make `{ identifierTypeUuid: string }` an actual interface type and add jsdoc comment on what that field does. (I admit I forgot to do that for `PatientAddressElementConfig`)

Reply:
Agreed.  Create a PatientIdentifierElementConfig that takes in an identiierTypeUuid, and could be expanded in the future.